src/AdminAcceptanceTests.Actions/Pages/CreateBuyingOrganisation.cs

```diff
@@ -58,6 +58,11 @@ public void ConfirmationPageDisplayed()
             Wait.Until(d => d.FindElements(Pages.CreateBuyingOrganisation.ConfirmationPage).Count > 0);
         }
 
+        public void OrganisationAlreadyExistsPageDisplayed()
+        {
+            Wait.Until(d => d.FindElements(Pages.CreateBuyingOrganisation.OrganisationAlreadyExistsPage).Count > 0);
```

Comment:
Is there anything specific on the page in particular we can wait for rather than a generic `error` page?

Reply:
It has got a specific data test ID